cfgov/core/tests/test_management_inactive_users.py

```diff
@@ -166,3 +163,43 @@ def test_sends_email_when_deactivating_and_warning(self):
                      stdout=self.stdout)
         # Outbox will have five user emails
         self.assertEqual(len(mail.outbox), 5)
+
+    def test_users_properly_deactivated(self):
+        """ Test that mail.EmailMessage is called with the appropriate
+        list of users """
+
+        mail.outbox = []
+        call_command('inactive_users',
+                     '--deactivate-users',
+                     stdout=self.stdout)
+
+        # Test that users were actually deactivated
+        # May need to re-fetch users
+        # inactive_users = User.objects.all()
+        User = get_user_model()
+        self.assertEqual(User.objects.get(username="user_1").is_active, False)
+        self.assertEqual(User.objects.get(username="user_2").is_active, False)
+        self.assertEqual(User.objects.get(username="üser_3").is_active, True)
+        self.assertEqual(User.objects.get(username="user_4").is_active, False)
+        self.assertEqual(User.objects.get(username="user_5").is_active, True)
+        self.assertEqual(User.objects.get(username="user_6").is_active, True)
+
+    def test_users_properly_deactivated_when_deactivating_and_warning(self):
+        """ Test that mail.EmailMessage is called with the appropriate
+        list of users """
+
+        mail.outbox = []
+        call_command('inactive_users',
+                     '--deactivate-users',
+                     '--warn-users',
+                     stdout=self.stdout)
+
+        # Test that users were actually deactivated
+        User = get_user_model()
+        self.assertEqual(User.objects.get(username="user_1").is_active, False)
+        self.assertEqual(User.objects.get(username="user_2").is_active, False)
+        self.assertEqual(User.objects.get(username="üser_3").is_active, True)
+        self.assertEqual(User.objects.get(username="user_4").is_active, False)
+        self.assertEqual(User.objects.get(username="user_5").is_active, True)
+        self.assertEqual(User.objects.get(username="user_6").is_active, True)
+
```

Comment:
Just a tiny-winy flake8 failure here.

Reply:
Piggybacking to add a minor style request: use `self.assertFalse` and `self.assertTrue` here instead of `self.assertEqual`.

Reply:
Roger! I'm looking at a post-it that says "Check whether `assertFalse` and `assertTrue` exist" 😆  Updating now.